shared/src/lib/utils.js

```diff
@@ -114,11 +114,18 @@ export async function fetchSettings() {
   };
 }
 
-export async function getActiveTab() {
-  const tabs = await browser.tabs.query({
+export async function getActiveTab(fetchingFromShortcut = false) {
+  const tabsQuery = {
     active: true,
     lastFocusedWindow: true,
-  });
+  };
+
+  // Don't look just in the last focused window if we're opening from the shortcut
+  if (fetchingFromShortcut) {
+    tabsQuery.lastFocusedWindow = undefined;
```

Comment:
When opening from the shortcut directly, sometimes the `lastFocusedWindow` would have no active tabs in Chromium (at least Chrome and Brave). This fixes that and it didn't change the behavior in Firefox.